cli/cmd/sync_v3.go

```diff
@@ -425,6 +426,16 @@ func syncConnectionV3(ctx context.Context, source v3source, destinations []v3des
 		Str("result", msg).
 		Msg("Sync summary")
 
+	analytics.TrackSyncFinished(ctx, invocationUUID, analytics.SyncFinishedEvent{
+		Source:        sourceSpec,
```

Comment:
I think we're pretty much interested in the plugin name (team name + plugin name), and the same for the destinations. That's how we track the syncs in Cloud as well.